main.py

```diff
@@ -28,7 +28,7 @@ def generate_batch(cols, rows):
     The first two elements are the image dimensions as specified in the document
     """
 
-    return [cols, rows] + [randint(0, 255) for _ in range(rows * cols)]
+    return [cols, rows] + [randint(0, 256) for _ in range(rows * cols)]
```

Comment:
This should be reverted to 255 due to randint being inclusive on both start and end